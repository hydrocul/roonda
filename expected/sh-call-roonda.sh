echo '[["abc", 10], ["def", 30]]' | $ROONDA_SELF_PATH --from-json --to-python2-obj-1
echo '[["abc", 10], ["def", 30]]' | $ROONDA_SELF_PATH --from-json --to-python2-obj-1
