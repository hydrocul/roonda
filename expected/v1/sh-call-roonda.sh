echo '[["abc", 10], ["def", 30]]' | $ROONDA_SELF_PATH --v1 --from-json --to-python2-obj
echo '[["abc", 10], ["def", 30]]' | $ROONDA_SELF_PATH --v1 --from-json --to-python2-obj
