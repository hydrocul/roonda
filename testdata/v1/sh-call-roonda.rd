sh v1

(pipe (echo "[[\"abc\", 10], [\"def\", 30]]")
      ((ref ROONDA_SELF_PATH) --v1 --from-json --to-python2-obj))

(pipe (echo "[[\"abc\", 10], [\"def\", 30]]")
      (roonda --v1 --from-json --to-python2-obj))

