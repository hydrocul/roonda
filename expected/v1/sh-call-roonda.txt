[["abc",10],["def",30]]
[["abc",10],["def",30]]
